Cargo.toml

```diff
@@ -42,6 +42,7 @@ rand_support = [
     "rand/getrandom",
 ]
 serde_support = ["serde"]
+fuzz_test = []
```

Comment:
Does `fuzz_test` not require `std` crate feature?

Reply:
good catch